data-model/SDP-sample-records/work_sample1.json

```diff
@@ -0,0 +1,34 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/work/10",
+  "label": "Hexaemeron (Basil of Caesarea)",
+  "entity_type": "work",
+  "uniform_name": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron",
+  "description": "A commentary on the six days of Creation. The first of its kind, this style of commentary, called Hexaemeron, flourished in late antiquity, and later commentaries drew on Basil's as a model.",
+  "authority_files": [{
+      "id": "https://haf.vhmml.org/work/320756617003",
+      "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron"
+    },
+    {
+      "id": "https://viaf.org/viaf/181934772",
+      "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. | Hexaemeron"
+    }
+  ],
+  "associated_dates": [{
+    "type": "creation",
+    "date_human_readable": "mid-4th c. CE",
+    "date_normalized": "0325/0375"
+  }],
+  "language": "Greek",
+  "genre": ["Theological Works", "Commentary, Biblical", "Commentary, Old Testament", "Commentary, Genesis"],
+  "associated_entities": [{
+      "type": "person",
+      "relation": "author",
+      "id": "https://sinaimanuscripts.library.ucla.edu/person/55",
+      "notes": []
+    }
+  ],
+  "witnesses": ["ark:/21198/dfdi53234x"],
```

Comment:
should this be associated_objects? Would need to distinguish between an associated textual artifact and a component of a textual artifact (i.e., if it's what the artifact embodies or if it appears in the TOC of an artifact). These would be backlinks anyway, but we do want to decide how to distinguish between 'has_witness' and 'is_part_of_witness'. This might also have implications for hierarchical relations between works, i.e. if a work appears in the TOC of an artifact, would we want to declare a relationship between the work records as well?